lang/src/jmh/kotlin/org/partiql/lang/PartiQLBenchmark.kt

```diff
@@ -0,0 +1,163 @@
+package org.partiql.lang
```

Comment:
This *must* not reside in this directory since that will be adding it to the `.jar` we publish to maven.  

This file should really reside in `lang/jmh/org/partiql/lang` instead of here.  To make this work, you need to modify the [`build.gradle`](https://github.com/partiql/partiql-lang-kotlin/blob/68e9f61e9a36090f2aeab80c560e786affa65845/build.gradle#L82-L92) in the root project by adding the lines noted below:

```
    plugins.withId('java', { _ ->
        sourceSets {
            main.java.srcDirs = ["src"]
            main.resources.srcDirs = ["resources"]
            test.java.srcDirs = ["test"]
            test.resources.srcDirs = ["test-resources"]

            // add these lines
            jmh.java.srcDirs = ["jmh"]
            jmh.resources.srcDirs = ["jmh-resources"]
        }        
    })
```

This may not be 100% exactly right but I'm certain something like this will be needed.

Reply:
I created the `jmh` sub-project but for some reason the benchmarks need to be under `./..src/jmh/kotlin/..` folder structure. I tried some other directory structures but the benchmark list wasn't populated for any other directory structure.